config/sets/symfony/annotations-to-attributes.php

```diff
@@ -18,4 +19,10 @@
         // @see Symfony 5.2+ https://github.com/symfony/doctrine-bridge/commit/02d2cf4743331e6b69ffd1d68e09b7e2dc417201#diff-1a16e2739e51eab000116d0542bd0226cea59a6d64711740ed7ce14769f95d1b
         new AnnotationToAttribute('Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity'),
     ]);
+    $rectorConfig->ruleWithConfiguration(
+        RenameClassRector::class,
+        [
```

Comment:
It seems not needed, just update the 2nd arg of `AnnotationToAttribute` arg to `Symfony\Component\Routing\Attribute\Route` above on line 18 should auto change to target attribute with new name

```php
new AnnotationToAttribute('Symfony\Component\Routing\Annotation\Route', 'Symfony\Component\Routing\Attribute\Route'
```

test is needed tho

Reply:
/cc  @scuben @JohJohan could you verify that "Symfony\\Component\\Routing\\**Attribute**\\Route" class exists in specific symfony version when upgrading from "Symfony\\Component\\Routing\\**Annotation**\\Route" thank you

Reply:
`Symfony\Component\Routing\Attribute\Route` is available since symfony 6.4 AFAIK

Reply:
That's what I mean, if that target class introduced in later version, eg transforming annotation is start from symfony 5.2, while the new target attribute class only exists since 6.4, then that RenameClassRector need to be placed in specific setlists, the issue is, that need to be checked that rename attribute, not rename annotation